source/reference/minio-mc/mc-get.rst

```diff
@@ -0,0 +1,92 @@
+==========
+``mc get``
+==========
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc get
+
+.. versionadded:: mc RELEASE.2024-02-24T01-33-20Z
+
+Syntax
+------
+
+.. start-mc-get-desc
+
+The :mc:`mc get` downloads an object from a target S3 deployment to the local file system.
+
+.. end-mc-get-desc
+
+Unlike other commands that can download files, such as :mc:`mc cp` or :mc:`mc mirror`, ``mc get`` is a simpler call.
+``mc get`` only does the one-way function of downloading the file so that it avoids the potential performance costs of other commands.
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following downloads the file ``logo.png`` from an s3 source to the local file system at path ``~/images/collateral/``.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc get minio/marketing/logo.png ~/images/collateral
+
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] get                      \
+                          SOURCE                   \
+                          TARGET                   \
+                          [--encrypt-key value]    \
+                          [--encrypt value] 
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: SOURCE
+   :required:
+
+   The full path to the :ref:`alias <minio-mc-alias>`, bucket, prefix (if used), and object of the path to download.
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The path on the local file system where the object should download to.
```

Comment:
An idea, perhaps not a great one, but maybe some variation on this:
```suggestion
   The destination path for the downloaded object on the local filesystem.
```

Reply:
I did some things to remove the ending preposition.